### DISCUSSION TITLE
Bug showing coin emoji on github

### DISCUSSION THREAD

**Discussion by tshemsedinov** [community/community#7081] [Other Feature Feedback, Questions, & Ideas]
**Place:** in *.md files and repository about block.
**Emoji code:** https://emojipedia.org/coin/
**See example:** https://github.com/metarhia/Metapay

![Screenshot from 2021-10-31 23-53-10](https://user-images.githubusercontent.com/4405297/139602452-3ea72b13-bed1-4727-b4f9-ceb66b8f34fc.png)

**Comment by Katsute** [comment]
I think that's an issue with the OS, the coin emoji doesn't exist on Windows 10 and below (it was added in Windows 11).

**Reply by tshemsedinov** [reply]
OS: Fedora 34
```
# uname -a
Linux marcus 5.13.19-200.fc34.x86_64 #1 SMP Sat Sep 18 16:32:24 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux
```

**Reply by tshemsedinov** [reply]
`coin` displays everywhere correctly in my system, even in terminal

**Reply by Katsute** [reply]
Then it's probably the browser font.
